mysql/errname.go

```diff
@@ -968,6 +968,21 @@ var MySQLErrName = map[uint16]string{
 	ErrColumnStateNonPublic:       "can not use non-public column",
 	ErrIndexStateCantNone:         "index %s can't be in none state",
 	ErrInvalidRecordKey:           "invalid record key",
+	ErrInvalidDDLWorker:           "Invalid DDL worker",
+	ErrUnsupportedDDLOperation:    "Unsupported",
+	ErrNotOwner:                   "TiDB server is not owner",
+	ErrCantDecodeIndex:            "Cannot decode index value, because %s",
+	ErrInvalidDDLJob:              "Invalid DDL job",
+	ErrInvalidJobFlag:             "Invalid job flag",
+	ErrWaitReorgTimeout:           "Timeout waiting for data reorganization",
+	ErrInvalidStoreVersion:        "Invalid storage current version: %d",
+	ErrUnknownTypeLength:          "Unknown length for type %d",
+	ErrUnknownFractionLength:      "Unknown length for type %d and fraction %d",
+	ErrInvalidJobVersion:          "Version %d of DDL job is greater than current (current what?) %d",
+	ErrInvalidSplitRegionRanges:   "Failed to split region ranges",
+	ErrReorgPanic:                 "Reorg worker panic",
+	ErrInvalidDDLState:            "Invalid %s state",
```

Comment:
```suggestion
	ErrInvalidDDLState:            "Invalid %s state %s",
```
e.g. `Invalid table state write only`.